src/lib/Output/Generator/InMemory/Xml/FieldTypeHashGenerator.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Rest\Output\Generator\InMemory\Xml;
+
+use Ibexa\Rest\Output\Generator\Json\FieldTypeHashGenerator as JsonFieldTypeHashGenerator;
+use Ibexa\Rest\Output\Generator\Json\JsonObject;
+
+final class FieldTypeHashGenerator extends JsonFieldTypeHashGenerator
```

Comment:
Same remark as before - there should be a common base from which each generator inherits. This would be a good opportunity to improve quality of `JsonFieldTypeHashGenerator` as well (e.g. strict types)

Reply:
Resolved in https://github.com/ibexa/rest/pull/121/commits/32ca57e103122bb065fec8317a3602f7d53cbd85